pos_order_to_sale_order/models/sale_order.py

```diff
@@ -12,31 +12,27 @@ class SaleOrder(models.Model):
     def _prepare_from_pos(self, order_data):
         PosSession = self.env["pos.session"]
         session = PosSession.browse(order_data["pos_session_id"])
+        SaleOrderLine = self.env["sale.order.line"]
+        order_lines = [
+            Command.create(SaleOrderLine._prepare_from_pos(line[2]))
+            for line in order_data["lines"]
+        ]
         return {
             "partner_id": order_data["partner_id"],
             "origin": _("Point of Sale %s") % (session.name),
             "client_order_ref": order_data["name"],
             "user_id": order_data["user_id"],
             "pricelist_id": order_data["pricelist_id"],
             "fiscal_position_id": order_data["fiscal_position_id"],
+            "order_lines": order_lines,
```

Comment:
```suggestion
            "order_line": order_lines,
```

https://github.com/odoo/odoo/blob/16.0/addons/sale/models/sale_order.py#L208

Should fix the CI

Reply:
@legalsylvain done